Makefile

```diff
@@ -204,6 +205,10 @@ ACTIONLINT := $(LOCALBIN)/actionlint-$(ACTIONLINT_VER)
 $(ACTIONLINT): | $(LOCALBIN)
 	$(call go-install-tool,$(ACTIONLINT),github.com/rhysd/actionlint/cmd/actionlint,$(ACTIONLINT_VER))
 
+WORKFLOWCHECK := $(LOCALBIN)/workflowcheck
+$(WORKFLOWCHECK): | $(LOCALBIN)
+	$(call go-install-tool,$(WORKFLOWCHECK),go.temporal.io/sdk/contrib/tools/workflowcheck,latest)
```

Comment:
Can/should we give it a specific version instead? Upgrading dependencies is a pain, but unpredictable behavior changes more so IMO.

Reply:
I think you're right! I changed that so it's using v0.3.0